### ISSUE TITLE
How did you organise training, validation and testing .tfrecord from ModelNet40?

### ISSUE THREAD

**Issue by Haodong-Alex-Yao** [daniilidis-group/spherical-cnn#10]
Hi,

Thanks for your great work!
Seems like in your training tfrecord files, you have around 468k instances while ModelNet40 only has 9843 instances. May I know how you generated these training, validation and testing data and what are the variables?

Best regards,
Alex

**Comment by machc** [comment]
Hi, thanks for your interest! Since pre-processing the meshes is slow, we save multiple augmented version of the dataset. Each of the training tfrecords should contain each element in the training set with some random rotations/anisotropic scaling/translation of the sphere center.  The original training set is split in train/validation, but for the final results we combine train and val. The test set also contains multiple rotated versions of each element on the original test set.

**Comment by Haodong-Alex-Yao** [comment]
Thanks for your reply Carlos,

If my understanding is correct, then the final result shows the SO3/ SO3 case. I suppose the most indicative value is the z/SO3 case. Do you have the training instances that only vary in Azimutal rotations in part of the tfrecord file?

Best regards,
Alex
